lang/Add_new_language.md

```diff
@@ -4,11 +4,11 @@ We will use Dutch as an example here.
 
 ## Prepare Prusa Firmware
 
-QR = palceholder for language in upper case
+QR = placeholder for language in upper case
 
 qr = placehodler for language in lower case
 
-AB = palceholder for hexadecial
+AB = placeholder for hexadecial
```

Comment:
:smile_cat: 3 words and all of them have/had typos 
- palceholder
- placehodler
- hexadecial

Reply:
Will update that in next PR and also add a workaround for non-block spaces generated by some editors.